lib/rubocop/config.rb

```diff
@@ -591,6 +591,8 @@ def check_target_ruby
             end
 
       msg += "\nSupported versions: #{KNOWN_RUBIES.join(', ')}"
+      
+      msg += "\nTry updatiing Rubocop to a newer version"
```

Comment:
You've got a typo here - updatiing. 

Also - it's RuboCop, not Rubocop. 

I'd suggest as a better wording "Newer RuboCop versions might support Ruby X." (X being the target Ruby versions")

Reply:
The above feedback has not been resolved.